backend/core/serializers.py

```
@@ -1413,25 +1413,151 @@ class TaskTemplateReadSerializer(BaseModelSerializer):
     assigned_to = FieldsRelatedField(many=True)
+
     next_occurrence = serializers.DateField(read_only=True)
     last_occurrence_status = serializers.CharField(read_only=True)
 
+    # Expose task_node fields directly
+    status = serializers.SerializerMethodField()
+    observation = serializers.SerializerMethodField()
+    evidences = serializers.SerializerMethodField()
+
     class Meta:
         model = TaskTemplate
```

Ruff (0.8.2)

`TaskTemplate` may be undefined, or defined from star imports

(F405)